adapters/repos/db/inverted/bm25_searcher.go

```diff
@@ -624,3 +623,16 @@ func (m AllMapPairsAndPropName) Less(i, j int) bool {
 func (m AllMapPairsAndPropName) Swap(i, j int) {
 	m[i], m[j] = m[j], m[i]
 }
+
+func PropertyIsSearchable(schemaDefinition *models.Schema, className, tentativePropertyName string) bool {
+	propertyName := strings.Split(tentativePropertyName, "^")[0]
```

Comment:
not sure about the context surrounding this. is `tentativePropertyName` guaranteed to always be at least `len() == 1` ?

Reply:
not sure TBH, it just is moved and renamed method from here https://github.com/weaviate/weaviate/blob/abf30eac8656cf3bf1b471d9eb1416695777a53d/entities/schema/accessors.go#L48-L63

pinging @donomii for answer 🙏